frontend/packages/knative-plugin/src/components/knative-resource/ConfigurationsOverviewList.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ListGroup } from 'patternfly-react';
+
+import { K8sResourceKind, referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink } from '@console/internal/components/utils';
+import { ConfigurationModel } from '@console/knative-plugin';
+
+type ConfigurationsOverviewListProps = {
+  configurations: K8sResourceKind[];
+};
+
+type ConfigurationsOverviewListItemProps = {
+  configuration: K8sResourceKind;
+};
+
+const ConfigurationsOverviewListItem: React.FC<ConfigurationsOverviewListItemProps> = ({
```

Comment:
Move all files related to overview from `src/components/knative-resource/*` to `src/components/overview` since we are already in the `knative-plugin`, adding `knative` is redundant`